services/galley/src/Galley/Data/Types.hs

```diff
@@ -43,6 +43,7 @@ data Conversation = Conversation
     , convTeam    :: Maybe TeamId
     , convDeleted :: Maybe Bool
     , convMessageTimer :: Maybe Milliseconds      -- ^ Global message timer
+    , convReceiptMode  :: Maybe ReceiptMode
```

Comment:
[not related to this PR, just a general observation]

- It would be helpful if this was called `ConversationDB` or something.  Then we wouldn't have to comment on it.
- I would prefer `ConversationMeta` to be `= ConverationMeta { cmCnv :: Conversation, cmMembers :: ... }` to eliminate redundancy.

But again, separate PR.  Just had so say something.  :-)

Reply:
> It would be helpful if this was called ConversationDB or something. Then we wouldn't have to comment on it.
> I would prefer ConversationMeta to be = ConverationMeta { cmCnv :: Conversation, cmMembers :: ... } to eliminate redundancy.

👍